waves-ext/src/test/scala/com/wavesplatform/dex/smart/MatcherScriptRunnerSpecification.scala

```diff
@@ -30,7 +30,7 @@ class MatcherScriptRunnerSpecification extends WavesExtSuiteBase {
     matcherFee = 30000L
   )
 
-  private def run(script: Script): Either[String, Terms.EVALUATED] = MatcherScriptRunner(script, sampleOrder)
+  private def run(script: Script): Either[String, Terms.EVALUATED] = MatcherScriptRunner(script, sampleOrder, isSynchronousCallsActivated = false)
```

Comment:
Hm.. could you add test with `isSynchronousCallsActivated = true` just to make sure all will work after a feature activation?